src/core/ddsc/tests/dispose.c

```diff
@@ -177,9 +177,11 @@ CU_TheoryDataPoints(ddsc_writedispose, non_writers) = {
 };
 CU_Theory((dds_entity_t *writer), ddsc_writedispose, non_writers, .init=disposing_init, .fini=disposing_fini)
 {
+    Space_Type1 data = { 0, 22, 22 };
     dds_return_t ret;
+
     DDSRT_WARNING_MSVC_OFF(6387); /* Disable SAL warning on intentional misuse of the API */
```

Comment:
Just a nitpick: I think these (this one and below) disable-warnings can be removed.

Reply:
Indeed. I think there are several more of those, I'll do that in a separate PR.